### PR TITLE
[M1 Mac environment] Try neuroconv specific debug

### PR DESCRIPTION
**Pull request by CodyCBakerPhD**
Trying a suggestion on your PR @rly

### CODE REVIEW

**Comment by CodyCBakerPhD** [comment]
NeuroConv branch/PR: https://github.com/catalystneuro/neuroconv/pull/563

**Comment by CodyCBakerPhD** [comment]
@rly I confirmed on my M1 I no longer need to do the uninstallation of sonpy step with this fix to NeuroConv

I'll update the branch pin once it's merged to `main` on NeuroConv

**Comment by CodyCBakerPhD** [comment]
> I'll update the branch pin once it's merged to main on NeuroConv

Upstream was merged, so updated here. `main` of NeuroConv should no longer install `sonpy` at all on M1

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "nwb-guide",
   "productName": "NWB GUIDE",
-  "version": "0.0.4",
+  "version": "0.0.5",
```
Comment (CodyCBakerPhD) [inline package.json:4]: I cut a new public release so @garrettmflynn could try reproducing bugs from the non-draft released deployments; as such we need to do a version bump to allow the next round of draft releases

**Review by rly — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (rly) [reply package.json:4]: Sounds good

Comment (garrettmflynn) [reply package.json:4]: @CodyCBakerPhD Just FYI Mac works great. Will try Windows later

Comment (garrettmflynn) [reply package.json:4]: Same behavior as observed in [#3](https://github.com/NeurodataWithoutBorders/nwb-guide/issues/354#issuecomment-1716559249), though the Electron issue linked seems to be relevant for development only if I'm not mistaken

**Pull request merged**